src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/config.properties

```diff
@@ -38,4 +38,5 @@ UsernameFieldName=User name field name
 WellknownConfigurationEndpoint=Well-known configuration endpoint
 UseRefreshTokens=Enable Token Refresh using Refresh Tokens
 DisableTokenExpirationCheck=Disable Token Expiration Check
-AllowedTokenExpirationClockSkewSeconds=Token Expiry Expiration Clock Skew
\ No newline at end of file
+AllowedTokenExpirationClockSkewSeconds=Token Expiry Expiration Clock Skew
+EnableTraditionalApiTokenAccess=Enable Traditional Jenkins API token access
```

Comment:
NIT: this to me is confusing, you are not enabling traditional Jenkins API token access here, you are mearly allowing its usage without a current OIdC Session.
I would suggest the name and text it changed to be something like `allowTokenAccessWIthoutSession` and "Allow Access with a Jenkins API token without an OIDC Session"